docs/source/technical_tutorials/search/maap_stac_r.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>It would be helpful to mention that these are the hrefs for the COGs with the biomass estimates!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>